### DISCUSSION TITLE
Tiled Mode on Selected (iDraw)

### DISCUSSION THREAD

**Discussion by tonypuglieso** [Orama-Interactive/Pixelorama#533] [Ideas]
https://user-images.githubusercontent.com/52177646/132243349-f6e77227-4aa8-41ca-b56c-07c6707329c0.mp4

Tile mode on selected and animation preview on selected
Spritesheet and tileset.

Hello, I come with a proposal inspired by an old pixel art program called Idraw Chara Maker, which was used for RPG Maker, you can download it for free to try these two functions. Download for trial http://tsukuru.pl/index.php?link=program/idraw (MODE compatible with Windows XP).

Example of a professional idraw chara maker user https://youtu.be/gYLWuQG3E6I?t=305

The Possible Solution Use The Extra View in Pixelorama, same as Idraw Chara Maker. Select with a rectangle, and in right Windows use the tile mode.

You use the Select Rectangle tool, then on what's selected, you can use the built-in pixelorama window on the right, to show the selection, and activate a kind of TiledMode. I would be reusing the code.

* Selection of rectangle.
* It only affects the selection.
* In the window on the right.
* See what is selected.
* Activate only in that window, the TiledMode.

Taking that into account. The second function may be integrated later.
Select multiple paintings in the square to see an animation.

This would be functional to work in Tiles and Spritesheets directly without having to reinvent the wheel, only with the tools that pixelorama already has.

It is a proposal of functions for pixelorama, the best pixel art program (although it already is: D).

I am aware that one cannot come to ask for functions because yes, there must be other priorities.

That is why, I understand that some GDScript, could be provided by the initial clues from where I can find the code of those functions (Select Rectangle, affect only selection, right view, Tiled mode) in the project. So I try to program those functions, even if it is a prototype for the future, it can serve the program.

Idraw reference

**Comment by Variable-ind** [comment]
Thanks for trying out pixelorama!!! Yeah maybe it could be done in the future :+1:

**Reply by Variable-ind** [reply]
Till then in version-0.9 (which is about to release in a few days and release candidate can be downloaded [HERE](https://nightly.link/Orama-Interactive/Pixelorama/workflows/dev-desktop-builds/master)) the selection system is now completely remade and an alternative to this can be achieved as follows:

https://user-images.githubusercontent.com/77773850/132245743-b3706e88-1b8e-4183-837c-4bb2b09ca5a6.mp4

**Reply by tonypuglieso** [reply]
Hello thanks for this video, now i'm on 0.9 beta (? on Ubuntu 20.04, In that version, for the moment I will work that way and in a future dream of having that integrated.
